themes/atomic/README.md

```diff
@@ -159,11 +159,18 @@ Three environment variables can be defined to rearrange the segments order. The
 
 `___ATOMIC_BOTTOM="char"`
 
-## Follow me
+### Contact / Social Media
 
-I'm on the social media.
+*Get the latest News about Web Development, Open Source, Tooling, Server & Security*
 
-* [@lfelipe1501](https://twitter.com/lfelipe1501) on Twitter.
-* [Luis Felipe](https://www.facebook.com/lfelipe1501) on Facebook.
+[![Twitter](https://github.frapsoft.com/social/twitter.png)](https://twitter.com/lfelipe1501)
+[![Facebook](https://github.frapsoft.com/social/facebook.png)](https://www.facebook.com/lfelipe1501)
+[![Github](https://github.frapsoft.com/social/github.png)](https://github.com/lfelipe1501)
+
+### Development by
+
+Developer / Author: [Luis Felipe Sánchez](https://github.com/lfelipe1501)
+
+Company: [lfsystems](https://www.lfsystems.com.co)
```

Comment:
...and neither is it a platform for advertising your own business.

Reply:
ok, I'm going to remove these lines